catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -153,10 +154,23 @@ private void updateIdentityNodeName(Metacard metacard)
                 extrinsicObjectType.setName(internationalStringTypeHelper.create(siteName)));
     Metacard identityMetacard = getRegistryMetacardFromRegistryPackage(registryPackage);
     if (identityMetacard != null) {
+      copyTransientAttributes(identityMetacard, metacard);
       federationAdminService.updateRegistryEntry(identityMetacard);
     }
   }
 
+  private void copyTransientAttributes(Metacard updateMetacard, Metacard existingMetacard) {
+    for (String transientAttributeKey : RegistryObjectMetacardType.TRANSIENT_ATTRIBUTES) {
+      Attribute transientAttribute = updateMetacard.getAttribute(transientAttributeKey);
+      if (transientAttribute == null) {
+        transientAttribute = existingMetacard.getAttribute(transientAttributeKey);
+        if (transientAttribute != null) {
+          updateMetacard.setAttribute(transientAttribute);
```

Comment:
❓ Should we check that the attribute does not already exist before setting it?

Reply:
We do it's just kind of hard to tell with how the method is written. I think the stream chris offered should fix that.